raycicmd/bk_pipeline.go

```diff
@@ -143,6 +143,7 @@ var (
 			map[string]any{"exit_status": 1, "limit": 1},
 			map[string]any{"exit_status": -1, "limit": 3},
 			map[string]any{"exit_status": 255, "limit": 3},
+			map[string]any{"exit_status": 53, "limit": 3},
```

Comment:
could you add a comment here on what 53 means? such as what failure example this is ?

we have the explanation for 125 and 127.

maybe also add the windows one too?